### PR TITLE
removed dependency on jaxb-api

### PR DESCRIPTION
**Pull request by alexradzin**

### CODE REVIEW

File: src/test/java/com/firebolt/jdbc/util/ByteArrayUtilTest.java
Comment (ptiurin) [inline src/test/java/com/firebolt/jdbc/util/ByteArrayUtilTest.java:1]: Let's add a couple more tests to be sure of quality. I'd suggest adding a test that parses empty string, single hex digit and invalid hex as well.

**Review by ptiurin — COMMENTED** [review]
Also, how much are we gaining here? how heavy is the jaxb-api package?

File: src/test/java/com/firebolt/jdbc/util/ByteArrayUtilTest.java
Comment (alexradzin) [reply src/test/java/com/firebolt/jdbc/util/ByteArrayUtilTest.java:1]: Good idea. Tests are added and code is fixed and improved. Tests rock!

**Comment by alexradzin** [comment]
> Also, how much are we gaining here? how heavy is the jaxb-api package?

https://packboard.atlassian.net/browse/FIR-29486

**Review by ptiurin — APPROVED** [review]

**Pull request merged**